generate.ts

```diff
@@ -106,22 +108,22 @@ function reportErrorsAndExit(errorMessages) {
 
     // Else, report all errors and exit the program
     console.log("Syntax: node generate.js\n" +
-        "\t\t(<targetName>=<targetOutputPath>|-destPath <destFolderPath>)\n" +
+        "\t\t(<templateName>=<outputPath>|-destPath <destFolderPath>)\n" +
```

Comment:
>outputPath [](start = 30, length = 10)

Is OutputPath functionally different from destFolderPath? Are they essentially the same?

Reply:
On that note, what do you think if we have this:

{ -templateName=<old-target-name> | -templatePath=<folder-containing-templates-for-some-private-repo>}

---
In reply to: [291384880](https://github.com/PlayFab/SDKGenerator/pull/546#discussion_r291384880) [](ancestors = 291384880)